crates/sui-node/src/metrics.rs

```diff
@@ -68,9 +68,9 @@ impl MetricsPushClient {
             network_key.private(),
             sui_tls::SUI_VALIDATOR_SERVER_NAME,
         ));
-        let identity = certificate.reqwest_identity();
+        let _identity = certificate.reqwest_identity(); // TODO enable for tls
         let client = reqwest::Client::builder()
-            .identity(identity)
+            // .identity(identity) // TODO enable for tls
             .build()
```

Comment:
These lines do not need to be changed and can be reverted. So long as the address provided is an http one, TLS and this identity won't be used.

Reply:
ok. i'll try that.  I thought I tried that already but i may not have.

Reply:
yep! turns out i left the https in the config when i tested it last time. reverted.